01-import_and_maxfilter.py

```diff
@@ -126,11 +127,47 @@ def run_maxwell_filter(subject, session=None):
         if hasattr(raw, 'fix_mag_coil_types'):
             raw.fix_mag_coil_types()
 
+        if config.find_flat_channels_meg or config.find_noisy_channels_meg:
+            if (config.find_flat_channels_meg and
+                    not config.find_noisy_channels_meg):
+                msg = 'Finding flat channels.'
+            elif (config.find_noisy_channels_meg and
+                  not config.find_flat_channels_meg):
+                msg = 'Finding noisy channels using Maxwell filtering.'
+            else:
+                msg = ('Finding flat channels, and noisy channels using '
+                       'Maxwell filtering.')
+
+            print(msg)
```

Comment:
we should consider using a proper logging system (later...)

Reply:
Absolutely!

Reply:
Opened an issue: #70